packages/shopify-codemod/transforms/remove-empty-catch-blocks.js

```diff
@@ -0,0 +1,16 @@
+export default function removeEmptyCatchBlocks({source}, {jscodeshift: j}, {printOptions = {}}) {
+
+  return j(source)
+    .find(j.TryStatement, {
+      type: 'TryStatement',
+      handler: {
+        type: 'CatchClause',
+      },
+    })
+      .forEach(({node}) => {
+        if (node.handler.body.body.length === 0) {
```

Comment:
Let's move this check into the filter of `find` (`body: {body: (body) => body.length === 0}` in the `handler` clause.